starfish/image/_stack.py

```diff
@@ -131,6 +133,82 @@ def set_slice(
         self._data[slice_list] = data
         self._data_needs_writeback = True
 
+    def show_stack(
+            self, indices: Mapping[Indices, Union[int, slice]], cmap: str='gray', rescale: bool=False,
```

Comment:
My personal newlining strategy is (in order of preference):
1. one line for parameters if it fits.  
2. split by logical groups.  this is a bit subjective, but in this case, self and indices might belong on one line, and the rest on the second line.
3. one line per parameter. (reduces the amount of lines highlighted when parameters are changed.

Feel free to disregard if you feel strongly otherwise.

Reply:
Also, unless cmap and figsize are very standard acronyms, I would go with color_map and figure_size.